### PR TITLE
windows: fix fs_write with nbufs > 1 and offset

### PR DESCRIPTION
**Pull request by unknownbrackets**
When multiple bufs are specified, overlapped needs to be advanced manually between each write.  Without this, each buf will be written to the same position (if offset is specified.)

[`ReadFile()`](http://msdn.microsoft.com/en-us/library/windows/desktop/aa365467%28v=vs.85%29.aspx) automatically advances, but [`WriteFile()`](http://msdn.microsoft.com/en-us/library/windows/desktop/aa365747%28v=vs.85%29.aspx) does not.

Since the buf flow seems to have been introduced only in unstable releases, I suppose no one should've been accidentally depending on the previous behavior.

-[Unknown]

### CODE REVIEW

**Comment by unknownbrackets** [comment]
Updated the commit with a test.

-[Unknown]

File: src/win/fs.c [hunk 1]
```diff
@@ -626,6 +622,13 @@ void fs__write(uv_fs_t* req) {
   index = 0;
   bytes = 0;
   do {
+    // WriteFile() does not advance overlapped as ReadFile() does.
```
Comment (saghul) [inline src/win/fs.c]: PLease use /\* */ style comments.

Comment (unknownbrackets) [inline src/win/fs.c]: Oops, right.  Been using C++ too much.

-[Unknown]

**Comment by saghul** [comment]
Thanks Ms/Mr Unknown! ;-) Landed in 6760d51

**Pull request closed without merging**